database/models.py

```diff
@@ -10,6 +10,7 @@ class Base(DeclarativeBase):
     id = Column(String(36), primary_key=True, default=lambda: str(uuid.uuid4()))
 
 
+# For ref: https://stackoverflow.com/questions/22698478/what-is-the-difference-between-the-declarative-base-and-db-model
```

Comment:
would be worthwhile to revisit this. is what we're currently doing not already handled by `flask-sqlalchemy`? no point in duplicating functionality